backend/apps/matching/management/commands/createfakeusers.py

```diff
@@ -29,30 +31,38 @@ def add_arguments(self, parser):
         )
 
         parser.add_argument(
-            "--add-a", nargs=1, help="Add [N] new students to the poll",
+            "--add-{a}".format(a=A.url_name),
```

Comment:
Lets use
```suggestion
            "--add-{a}".format(a=A.url_name.lower()),
```

Reply:
It's `url_name = name.lower()` in `A.py` and `B.py`

Reply:
Ah, now i get it.